github/login.go

```diff
@@ -57,8 +60,14 @@ func githubHandler(config *oauth2.Config, success, failure http.Handler) http.Ha
 			failure.ServeHTTP(w, req.WithContext(ctx))
 			return
 		}
+
 		httpClient := config.Client(ctx, token)
-		githubClient := github.NewClient(httpClient)
+		githubClient, err := githubClientFromAuthURL(config.Endpoint.AuthURL, httpClient)
+		if err != nil {
+			ctx = gologin.WithError(ctx, fmt.Errorf("could not parse AuthURL %s", config.Endpoint.AuthURL))
```

Comment:
The error message at this stage should be about the call being made (we don't know its a parse error at this level and shouldn't assume). Also for consistency with the rest of this package, there's the `github:` prefix.

Perhaps `fmt.Errorf("github: failed to create github Client: %v", err)`